test/unit/terrain/terrain.test.js

```diff
@@ -263,6 +265,96 @@ test('Elevation', (t) => {
         });
     });
 
+    t.test('map._updateAverageElevation', t => {
+        const map = createMap(t, {
+            style: extend(createStyle(), {
+                layers: [{
+                    "id": "background",
+                    "type": "background",
+                    "paint": {
+                        "background-color": "black"
+                    }
+                }]
+            })
+        });
+        map.setPitch(85);
+        map.setZoom(13);
+
+        map.once('style.load', () => {
+            map.addSource('mapbox-dem', {
+                "type": "raster-dem",
+                "tiles": ['http://example.com/{z}/{x}/{y}.png'],
+                "tileSize": TILE_SIZE,
+                "maxzoom": 14
+            });
+            map.transform._horizonShift = 0;
```

Comment:
I see this line in a few similar tests. What effect does it have here? Does it just unset a safety fudge factor?

Reply:
When we merged an optimization for horizon shift to reduce some tiles that have low screen area coverage, a lot of unit tests failed here, so to maintain legacy behavior in tests without having to update a lot of small diff of floating point values, we forced horizon shift to be 0.

This was incorrectly carried over when I copied the skeleton of the test, so I just removed it. Thanks for the catch.